### PR TITLE
Rename vaTreatments -> treatments

### PR DESCRIPTION
**Pull request by cvalarida**

### CODE REVIEW

File: script/run-unit-test.sh [hunk 1]
```diff
@@ -14,7 +14,7 @@ if [[ -z "$1" ]] || [[ $1 == -* ]]; then
   exit $?
 fi
 
-BABEL_ENV=test $MOCHA src/platform/testing/unit/helper.js "$@"
+BABEL_ENV=test $MOCHA --opts src/platform/testing/unit/mocha.opts src/platform/testing/unit/helper.js "$@"
```
Comment (markgreenburg) [inline script/run-unit-test.sh]: Does this maybe warrant its own PR?

File: test/disability-benefits/526EZ/config/vaFacilities.unit.spec.jsx [hunk 2]
```diff
@@ -38,23 +38,23 @@ describe('Disability benefits 526EZ VA facility', () => {
...
     );
 
-    fillData(form, 'select#root_vaTreatments_0_treatmentDateRange_fromMonth', '1');
-    fillData(form, 'select#root_vaTreatments_0_treatmentDateRange_fromDay', '3');
-    fillData(form, 'input#root_vaTreatments_0_treatmentDateRange_fromYear', '1950');
-    fillData(form, 'select#root_vaTreatments_0_treatmentDateRange_toMonth', '1');
-    fillData(form, 'select#root_vaTreatments_0_treatmentDateRange_toDay', '3');
-    fillData(form, 'input#root_vaTreatments_0_treatmentDateRange_toYear', '1955');
-    fillData(form, 'input#root_vaTreatments_0_treatmentCenterName', 'Local facility');
+    fillData(form, 'select#root_treatments_0_treatmentDateRange_fromMonth', '1');
```
Comment (markgreenburg) [inline test/disability-benefits/526EZ/config/vaFacilities.unit.spec.jsx]: For what it's worth, I had trouble when I tried switching these to `fillDate` and I didn't have the time to investigate. Wondering if you ran into issues as well or if you just decided to leave it as-is for the sake of keeping the PR focused?

**Review by markgreenburg — APPROVED** [review]
LGTM! Wondering if the testing script update might fit better in its own PR though.

File: test/disability-benefits/526EZ/config/vaFacilities.unit.spec.jsx (hunk 2 above)
Comment (cvalarida) [reply test/disability-benefits/526EZ/config/vaFacilities.unit.spec.jsx]: Yeah, I didn't try. Suppose I could give it a whirl.

File: script/run-unit-test.sh (hunk 1 above)
Comment (cvalarida) [reply script/run-unit-test.sh]: Oh, yup. That wasn't supposed to sneak in here.

**Review by cehsu — APPROVED** [review]
LGTM.

**Pull request merged**